test/gtest/cpu_multi_head_attention.hpp

```diff
@@ -105,8 +131,8 @@ struct CPUMHATest : public ::testing::TestWithParam<CPUMHATestCase>
 
         const auto [error, max_diff] = calcStats(final_transformed_attention);
         // CI clang-tidy treats is as "boolean value assigned to float"
-        const double error_threshold    = ((std::is_same_v<OutputType, float>) ? 1e-7 : 1e-2);
-        const double max_diff_threshold = ((std::is_same_v<OutputType, float>) ? 1e-6 : 1e-1);
+        const double error_threshold    = ((std::is_same_v<OutputType, float>) ? 1e-7 : 5e-2);
```

Comment:
is the first one relative error? or what is the difference between the two?

Reply:
I think one is for float and other is for fp8? Since fp8 has lower precision hence the lower tolerance.

Reply:
If you mean different thresholds for the errors - @bghimireamd has already answered.
If you mean error and max_diff then the first one is rms and the second one is maximum absolute difference (
`sqrt(sum((x1-y1)^2...) / n)` and `max(abs(x1-y1)...)`)
Since rms hides outliers, I'd like to control maximum difference as well.

Reply:
@CAHEK7 : thanks. That's what I was looking for.